build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -285,6 +285,12 @@ flowExporter:
   # packet matching this flow has been observed since the last export event.
   # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
   idleFlowExportTimeout: {{ .idleFlowExportTimeout | quote }}
+
+  # Optional: Provide the set of protocols to filter specific flows that will be
+  # exported. By default, all protocols supported by OVS are exported. Invalid protocols
+  # will be not error and instead a warning is logged on the antrea agent. Supported
+  # protocols include: "TCP", "UDP", "SCTP"
```

Comment:
I have a preference for lower-case here, I would also make sure that we accept values irrespective of the case